app/src/main/java/org/liberty/android/fantastischmemo/ui/StatisticsScreen.java

```diff
@@ -168,6 +172,50 @@ public void onPostExecute(ResultT result) {
         }
     }
 
+    private class NewCardTask extends ChartTask<Void, Void, BarData> {
+
+        public static final int INITIAL_CAPACITY = 30;
+
+        @Override
+        public BarData doInBackground(Void... params) {
+            cardDao = dbOpenHelper.getCardDao();
+            Integer DAYS = 30;
+            List<String> xVals = new ArrayList<String>(INITIAL_CAPACITY);
+            List<BarEntry> yVals = new ArrayList<BarEntry>(INITIAL_CAPACITY);
+            Integer MILLISECONDS_PER_DAY = 60 * 60 * 24 * 1000;
+            Date now = new Date();
+            for (int i = -INITIAL_CAPACITY; i < 1; i++) {
+                Date endDate = new Date(now.getTime() + i * MILLISECONDS_PER_DAY);
+                Date startDate = new Date(endDate.getTime() - MILLISECONDS_PER_DAY);
+                xVals.add("" + i);
+                yVals.add(new BarEntry((float)cardDao.getNewLearnedCardCount(null, startDate, endDate),
+                                       INITIAL_CAPACITY + i)); // the order has to be nonnegative
+
+            }
+
+            BarDataSet dataSet = new BarDataSet(yVals, getString(R.string.number_of_new_cards_learned_in_a_day_text));
+            BarData data = new BarData(xVals, dataSet);
+            data.setValueTextColor(Color.WHITE);
+
+            return data;
+
+        }
+
+        @Override
+        public Chart generateChart(BarData data) {
+            BarChart chart = new BarChart(StatisticsScreen.this);
+            chart.setDrawGridBackground(false);
+            chart.getLegend().setTextColor(Color.WHITE);
+            chart.getXAxis().setTextColor(Color.WHITE);
+            chart.getAxisLeft().setTextColor(Color.WHITE);
+            chart.getAxisRight().setTextColor(Color.WHITE);
+
+            chart.setData(data);
+            chart.setDescription("");
```

Comment:
I intensionally left out the description here for the consistency with other plotting methods. And it doesn't look too good when we add it to the chart, since the description would overlap with the bar (I would certainly send you a screenshot if you would like to have a look). 

The description is actually already included in the legends. 

So I was wondering if it was just a misunderstanding.